au/code/au/unit_of_measure.hh

```diff
@@ -818,6 +818,19 @@ struct CommonOrigin<Head, Tail...> :
                                OriginOf<Head>,
                                CommonOrigin<Tail...>>>> {};
 
+// `UnitOfLowestOrigin<Us...>` is any unit among `Us` whose origin equals `CommonOrigin<Us...>`.
+template <typename... Us>
+struct UnitOfLowestOriginImpl;
+template <typename... Us>
+using UnitOfLowestOrigin = typename SortAs<UnitProduct, UnitOfLowestOriginImpl<Us...>>::type;
```

Comment:
I see it's a pattern already in the file, so more of a discussion point:  I generally name the type to be what it is (`U`), as opposed to encoding the fact that it's more than one in the type name (`Us`).  Would it always be obvious from context that `U` was 0, 1, or more items (i.e. a pack)?